content/get-started/writing-on-github/working-with-advanced-formatting/organizing-information-with-collapsed-sections.md

```diff
@@ -43,6 +43,26 @@ After a reader clicks {% octicon "triangle-right" aria-label="The right triangle
 
 ![Screenshot of the Markdown above on this page as rendered on {% data variables.product.prodname_dotcom %}, indicating that a collapsed section can contain headers, sentences of text, images, and code blocks.](/assets/images/help/writing/open-collapsed-section.png)
 
+## Creating a collapsible section that is open by default
+
+````markdown
+<details open>
+
+<summary>Tips for collapsible sections that are open by default</summary>
+
+### You can add a header
+
+You can add text within a collapsed section. 
+
+You can add an image or a code block, too.
+
+```python
+   print("Hello World")
+```
+
+</details>
+````
```

Comment:
```suggestion
Optionally, to make the section display as open by default, add the `open` attribute to the `<details>` tag:

```html
<details open>
```

Suggestions don't play nicely with other triple backticks, so you'll have to add three to close the `html` environment.

Reply:
Hello again @nguyenalex836 :wave: 

Hello @subatoi 
Oh dang, somehow I didn't manage to find that Quickstart page. :disappointed:
Apologies for the lack of time on the issue ticket.
I agree it is best to keep it simple and I agree with your suggestion.
Thank you!

There are lots of docs. Corral them and hyperlink 'em, right? :sweat_smile:
I'm wiser to the next time and wish you all the best in arranging/revising the GH docs. :saluting_face: